src/rpc/client.cpp

```diff
@@ -110,6 +110,11 @@ static const CRPCConvertParam vRPCConvertParams[] =
     { "combinerawtransaction", 0, "txs" },
     { "fundrawtransaction", 1, "options" },
     { "fundrawtransaction", 2, "iswitness" },
+    { "walletupdatepsbt", 2, "psbtformat"},
```

Comment:
I think the `wallet` prefixes are helpful

Reply:
> I think the wallet prefixes are helpful

>I prefer the wallet prefixes and expanding out psbt to partiallysignedtransaction just makes the RPC names too long IMO.

We now have several wallet RPC methods that are not prefixed with `wallet`, so IMO is an unnecessary prefix. 

Also, by the spec `PSBT` is a transaction format, so the transaction itself is a `partially signed transaction`, or `pst`, so using `psbt` as suffix somehow seems improper.

Given that, WDYT about not using the prefix `wallet`, use the `ps` (term for `partially signed`) as a transaction type, and use the suffix `transaction`:
- `walletcreatepsbt` to `createpstransaction`
- `walletupdatepsbt` to `updatepstransaction`
- `combinepsbt` to `combinepstransactions`
- `decodepsbt` to `decodepstransaction`

Reply:
I think it's the other way around. Historically no RPCs had a wallet prefix, which led to a lot of confusion (e.g. gettransaction vs getrawtransaction).

I'm in favor of having wallet in the name of RPCs that require a wallet to interact with.

Reply:
`bip174-create`
`bip174-update`
`bip174-combine`
`bip174-decode`

Reply:
I agree with @sipa and prefer to have the wallet prefix, especially because there will probably be a non-wallet version of those RPCs at some point in the future.

I don't think psbt is really that ambiguous..

Reply:
> I think it's the other way around. Historically no RPCs had a wallet prefix, which led to a lot of confusion (e.g. gettransaction vs getrawtransaction).

Agree on that. If all methods will be prefixed LGTM.